onnxruntime/test/perftest/main.cc

```diff
@@ -9,6 +9,9 @@
 
 using namespace onnxruntime;
 
+const OrtApi* g_ort = OrtGetApi(ORT_API_VERSION);
+const OrtApi* Ort::g_api = OrtGetApi(ORT_API_VERSION);
```

Comment:
So we should not forget to initialize one global for macros in the CXX APi header and one for some other purpose?
I do not like the gist of it. It would be nice to have an instance of a class that does it and forget about it.
Adjust macros in a way that they don't rely on a global extern. We should get rid of externs.

Reply:
I don't like that part either, and I did adjust the macros. Or do you mean the inline functions?